reconcile/cli.py

```diff
@@ -1458,11 +1458,11 @@ def gabi_authorized_users(ctx, thread_pool_size, internal, use_jump_host):
                     use_jump_host)
 
 
-@integration.command()
-@enable_deletion(default=False)
-@click.pass_context
-def dyn_traffic_director(ctx, enable_deletion):
-    run_integration(reconcile.dyn_traffic_director, ctx.obj, enable_deletion)
+# @integration.command()
+# @enable_deletion(default=False)
+# @click.pass_context
+# def dyn_traffic_director(ctx, enable_deletion):
```

Comment:
Is removing this intended? Not mentioned in the PR description

Reply:
no no, it was a mistake